modules/auxiliary/scanner/portscan/tcp.rb

```diff
@@ -17,6 +17,11 @@ class Metasploit3 < Msf::Auxiliary
   def initialize
     super(
       'Name'        => 'TCP Port Scanner',
+      'Description' => %q{
```

Comment:
There are now two `Description` values in the info hash which will likely result in a warning from Ruby.  In the interest of keeping this PR focused, I'd suggest just removing this change.